build/release/release.sh

```diff
@@ -98,52 +98,28 @@ upload_svn_staging() {
 
 upload_nexus_staging() {
   echo "Deploying celeborn-client-spark-2-shaded_2.11"
-  ${PROJECT_DIR}/build/mvn clean install -DskipTests -Papache-release,spark-2.4 \
-    -s "${PROJECT_DIR}/build/release/asf-settings.xml" \
-    -pl :celeborn-client-spark-2-shaded_2.11 -am
-  ${PROJECT_DIR}/build/mvn deploy -DskipTests -Papache-release,spark-2.4 \
-    -s "${PROJECT_DIR}/build/release/asf-settings.xml" \
-    -pl :celeborn-client-spark-2-shaded_2.11
+  ${PROJECT_DIR}/build/sbt -Pspark-2.4 "clean;celeborn-client-spark-2-shaded/publish"
 
   echo "Deploying celeborn-client-spark-3-shaded_2.12"
-  ${PROJECT_DIR}/build/mvn clean install -DskipTests -Papache-release,spark-3.3 \
-    -s "${PROJECT_DIR}/build/release/asf-settings.xml" \
-    -pl :celeborn-client-spark-3-shaded_2.12 -am
-  ${PROJECT_DIR}/build/mvn deploy -DskipTests -Papache-release,spark-3.3 \
-    -s "${PROJECT_DIR}/build/release/asf-settings.xml" \
-    -pl :celeborn-client-spark-3-shaded_2.12
+  ${PROJECT_DIR}/build/sbt -Pspark-3.4 "clean;celeborn-client-spark-3-shaded/publish"
```

Comment:
align with the script `build/make-distribution.sh`

https://github.com/apache/incubator-celeborn/blob/089a0f868600f4ea280c63cc0960586bf9389b55/build/make-distribution.sh#L308